src/views/OnChainProperty.vue

```diff
@@ -10,7 +11,20 @@
 		      :size="icons[index].size"
 		      :pack="icons[index].pack"
 		      :icon="icons[index].icon"
-		  		@click.native="toast('Copied to clipboard')"
+		  		@click.native="toast('Copied to clipboard', icons[index])"
```

Comment:
so copy to clipboard and open new window 🤔 I mean get it might be interesting to copy to clipboard if someone wants to share that URL to twitter. but for now having that buttons on right copy to URL of artists profile is enough I guess :)

Reply:
That second parameter should be removed, sorry. I'm not sure what you mean though, my thought was that if the button is not Twitter than it maintains the previous behavior and if it is Twitter it navigates to the URL. Handled by this logic: 		 v-if="icons[index].icon !== 'twitter'"
So it will not copy to clipboard if it is the Twitter button.